src/src/WebServer/WebServer.cpp

```diff
@@ -542,13 +542,16 @@ void getWebPageTemplateDefaultHead(String& tmpl, bool addMeta, bool addJS) {
             "</head>");
 }
 
+# define GITHUB_RELEASES_LINK_PREFIX "<a href='https://github.com/letscontrolit/ESPEasy/releases' title='Click to show latest release on Github' target='_blank' style='font-size: 15px; text-decoration: none'>"
+# define GITHUB_RELEASES_LINK_SUFFIX "</a>"
```

Comment:
Maybe wrap these in `#ifndef` checks, so these can also be customized in `Custom.h`

Reply:
Moved to `ESPEasyDefaults.h` and made conditional